### PR TITLE
Deploy to Production (12th Nov 2021)

### PR DESCRIPTION
**Pull request by paphonb**
# [Related Task]()

## Link for [Demo]()

- (Optional) Link for [Figma]()
- (Optional) Other [Resources]()

## Why do you do this PR

-

## What did you do

-

## Checklist

- [ ] Merged this PR into [dev](https://github.com/thinc-org/cugetreg-frontend/tree/dev) for Demo
- [ ] Wrote coverage tests

## Browser Supported

- [x] Safari
- [x] Google Chrome
- [x] Firefox
- [x] Microsoft Edge

## Other thing to tell us

-

### CODE REVIEW

**Review by saenyakorn — APPROVED** [review]

**Comment by paphonb** [comment]
Let's wait for #260 before merging this

File: src/common/utils/getCourseKeyString/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { CourseKey } from '@/common/utils/types'
+
+export function getCourseKeyString(course: CourseKey): string {
+  return `${course.courseNo}-${course.studyProgram}-${course.academicYear}-${course.semester}`
```
Comment (new5558) [inline src/common/utils/getCourseKeyString/index.ts:4]: Will this face same problem as #260 ?

File: src/modules/Schedule/components/Schedule/index.tsx [hunk 2]
```diff
@@ -27,10 +27,7 @@ function Schedule({ classes }: ScheduleProps) {
       <Header />
       <Gutters />
       {classes.map((scheduleClass) => (
-        <ClassCard
-          key={`${scheduleClass.courseNo}_${scheduleClass.dayOfWeek}_${scheduleClass.position.start}`}
-          scheduleClass={scheduleClass}
-        />
```
Comment (new5558) [inline src/modules/Schedule/components/Schedule/index.tsx:33]: Will key error by same reason as #260

**Comment by new5558** [comment]
LGTM if not having key error problems

**Comment by new5558** [comment]
https://beta.cugetreg.com/S/courses/0123104
Still internal server error

**Review by new5558 — CHANGES_REQUESTED** [review]
https://beta.cugetreg.com/S/courses/0123104
Still internal server error

File: src/modules/Schedule/components/Schedule/index.tsx (hunk 2 above)
Comment (paphonb) [reply src/modules/Schedule/components/Schedule/index.tsx:33]: This is what #260 fixed

File: src/common/utils/getCourseKeyString/index.ts (hunk 1 above)
Comment (paphonb) [reply src/common/utils/getCourseKeyString/index.ts:4]: No. There will be only one course card per course.

**Comment by paphonb** [comment]
> https://beta.cugetreg.com/S/courses/0123104
> Still internal server error

#261 should fix this

**Review by new5558 — APPROVED** [review]

**Pull request merged**